src/project.rs

```diff
@@ -931,16 +930,9 @@ pub struct Manifest {
 }
 
 impl Manifest {
-    pub fn parse(path: &Path) -> Result<Self> {
+    pub fn parse() -> Result<Self> {
         let metatdata = MetadataCommand::new().exec()?;
-        let package = metatdata
-            .packages
-            .iter()
-            .find(|p| p.manifest_path == path)
-            .expect({
-                let path = path.display();
-                &format!("could not find package for {}", path)
-            });
+        let package = metatdata.packages.first().expect("at least one package");
```

Comment:
Again, it isn't clear to me that this is something we can `expect` and shouldn't gracefully produce and propagate an error when we run into?

Reply:
https://github.com/rust-fuzz/cargo-fuzz/pull/349/commits/3a91261ea379943c256ea022f83e418a614a634f